README.md

```diff
@@ -1,2 +1,5 @@
 # Asking
 Iam thinking about programming a bot who deleted spam automaticcly is this good idea?
+
+# Answer
+Yes very good idea
```

Comment:
Can you please provide additional data points to support this conclusion? I am specifically looking for information related to what the value is for potential customers. A cost analysis and a path to profitability would go a long way here.

Reply:
Sir this is a Wendy's

Reply:
ok i will start working on it this week if i get any problems or news i will ask you/ send you the beta version